docs/integrations/amazon-aws/ses.md

```diff
@@ -258,7 +258,7 @@ This section provides examples of the Amazon SES App dashboards, along with desc
 
 See the overview of SES CloudTrail events including failed, and successful events, error codes, users, and event locations.
 
-<img src={useBaseUrl('img/integrations/amazon-aws/AmazonSESCloudTrailEventsbyEventName.png')} alt="AWS API Gateway" />
+<img src={useBaseUrl('img/integrations/amazon-aws/AmazonSESCloudTrailEventsOverview.png')} alt="AWS SES CloudTrail Events Overview" />
```

Comment:
It is broken when previewed, kindly check.

Reply:
Adding the image.